go.mod

```diff
@@ -1,5 +1,5 @@
-module github.com/NandosUK/talon_go/v7
+module github.com/talon-one/talon_go/v8
```

Comment:
This changed however for the sake of continuity and forward compatibility with talon-one's version of the library we should be using the replace directive

https://go.dev/doc/modules/gomod-ref#replace

Reply:
Yeah it should since the import path is already talon_go. I'll double check before this merges

https://github.com/NandosUK/promotions-proxy/blob/c7bfe7ccfb20f13750d2024bbf37b574cbd22807/internal/provider/talon/processors/discounts/process_custom_effect.go#L6